llvm/lib/Target/SPIRV/SPIRVEmitIntrinsics.cpp

```diff
@@ -174,7 +174,7 @@ static bool isAggrToReplace(const Value *V) {
 
 static void setInsertPointSkippingPhis(IRBuilder<> &B, Instruction *I) {
```

Comment:
Maybe you want getInsertionPointAfterDef()?

The exact way you write it doesn't really matter much if you don't have exception-handling, though.

Reply:
Not exactly, it's a bit different case. But thank you very much for the pointer, `getInsertionPointAfterDef()` and its applications were worth a study.